### PR TITLE
Xcorner 23 blog post page

### PR DESCRIPTION
**Pull request by mgbelegu**

### CODE REVIEW

File: assets/scss/components/_article.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    font-style: normal;
+    font-weight: 600;
+    font-size: rem-calc(32px);
+    line-height: rem-calc(40px);
+    margin-bottom: rem-calc(12px);
+
+    @include bp(medium) {
+        font-size: rem-calc(42px);
+        line-height: rem-calc(56px);
+        margin-bottom: rem-calc(31px);
```
Comment (marcinkrzeminski) [inline assets/scss/components/_article.scss]: @mgbelegu please set it to 30. None is going to notice this 1px ;).

File: assets/scss/components/_article.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    line-height: rem-calc(24px);
+    margin-bottom: rem-calc(20px);
+
+    @include bp(medium) {
+        margin-bottom: rem-calc(50px);
+    }
+}
+
+.c-blog-article__container {
+    max-width: rem-calc(782px);
```
Comment (marcinkrzeminski) [inline assets/scss/components/_article.scss]: @mgbelegu I'm not sure about this one as well. If `780` doesn't break anything, let's use it.

**Review by marcinkrzeminski — CHANGES_REQUESTED** [review]
@mgbelegu just two tiny comments.

**Comment by mgbelegu** [comment]
Done :) will merge

**Pull request merged**